### ISSUE TITLE
Consider renaming the final font files

### ISSUE THREAD

**Issue by kytta** [evilmartians/mono#27]
Hi! Love the font! However, I am having issues with the naming of the fonts. As of now, the names are abbreviated:

<img width="196" alt="Screenshot of the Fonts.app with the names of font weights of Martian Mono" src="https://user-images.githubusercontent.com/6566248/215732372-5624a3d2-c9de-4495-9a07-29045d91a092.png">

This makes them hard to parse, at least for me: I wouldn't expect ‘Light’ to be actually called ‘Std Lt’. Some applications, though, also have issues with it. Example use case: In JetBrains’ editors, I use a 14pt font for code editing and a 12pt font in my terminal. For this, I have to uncheck the ‘Use default font’ checkbox. When I do it, I don't have the option to set the font weight, only the family. Since IntelliJ can't find the ‘Normal’ weight, it defaults to the first font in alphabetical order, it being ‘Condensed Bold’:

<img width="1094" alt="Screenshot of PyCharm settings with a wrong font being selected" src="https://user-images.githubusercontent.com/6566248/215733735-3d723a63-c25c-4a80-a47e-21e0604acbd1.png">

I know, this is rather an IntelliJ problem for not letting one choose the font weight there, but it's also a font problem IMO, since there is a somewhat established tradition in how to name the font weights.

Is there a reason for the shorter names being used, and if no, would you consider renaming them in the next release?

**Comment by romashamin** [comment]
Hey @kytta! Thank you for your question 🙌

You can switch to the [Google Fonts version](https://fonts.google.com/specimen/Martian+Mono/glyphs) of the Martian Mono font. It has the naming convention you’re asking for.

The current naming system—based on abbreviations—was a conscious choice.

I understand why you expected just _Light_—because you’ve worked with single-dimension fonts, which include only one Light font style. However, in multi-dimensional fonts, there are several Light styles: Condensed Light, Narrow Light, Standard Light, and so on. Moreover, variable fonts reconsider the whole concept of font styles.

Anyway, let’s get back to the naming.

If the font designer, working on a large font family, decides to use the full style names, they inevitably end up with very long names.

For example:

```
Martian Mono Semi Expanded Extra Bold Italic
Martian Mono Condensed Extra Bold Italic
...
```

This name may be a little easier for understanding. At first. But after you’ll notice that some apps (like Adobe) cut the long names. And this is what the user sees:

```
Martian Mono Semi Expan...
Martian Mono Semi Expan...
Martian Mono Semi Expan...
```

I’m trying to say that there’s no silver bullet. Every name convention has its own pros and cons. Back then I considered all of them and made my decision in favor of short abbreviations.

Finally, take into consideration that the problem is wider than just font style names. Actually, the way our software lists the font styles also matter. Take a look at the image.

<img width="504" alt="font-style-naming" src="https://user-images.githubusercontent.com/1148914/216787821-cdb05c2a-aeec-4c80-a386-26b8d18cb9be.png">

Notice how easier it is to get an idea of the font naming if the software breaks font styles into groups and keeps them in order as in the example on the right.

*  *  *

It’s sad to hear that JetBrains—the industrial standard of apps for development—is unable to implement in their IDEs the proper support for variable fonts, and decimal font sizes, and improve the whole user experience around working with fonts.

As I said, consider switching to the GF version of Martian Mono.

**Comment by kytta** [comment]
Hello Roman,

thank you a lot for taking the time to write such a detailed response!

> I understand why you expected just _Light_—because you’ve worked with single-dimension fonts, which include only one Light font style. However, in multi-dimensional fonts, there are several Light styles: Condensed Light, Narrow Light, Standard Light, and so on. Moreover, variable fonts reconsider the whole concept of font styles.

Yeah, you're right. 😄 I am used to fonts that have the full names, even if they're multidimensional. But I understand that `Martian Mono Semi Expanded Extra Bold Italic` is not æsthetically pleasing by any means :D
 
> Finally, take into consideration that the problem is wider than just font style names. Actually, the way our software lists the font styles also matter. Take a look at the image.

Damn, I wish every app did this!

> As I said, consider switching to the GF version of Martian Mono.

Yup, this works for me. Thanks!
